src/Redux/admins/reducer.js

```diff
@@ -0,0 +1,88 @@
+import {
+  GET_ADMINS_PENDING,
+  GET_ADMINS_SUCCESS,
+  GET_ADMINS_ERROR,
+  GET_ADMINBYID_PENDING,
```

Comment:
Note that this type of constant must respect the 'GET_ADMINS_BY_ID_PENDING' convention.